rust/crd/src/user_info_fetcher.rs

```diff
@@ -115,12 +115,12 @@ pub struct ActiveDirectoryBackend {
     pub custom_attribute_mappings: BTreeMap<String, String>,
 }
 
-#[derive(Clone, Debug, Deserialize, Eq, JsonSchema, PartialEq, Serialize, Derivative)]
-#[derivative(Default)]
+#[derive(Clone, Debug, Deserialize, Eq, JsonSchema, PartialEq, Serialize, Educe)]
+#[educe(Default)]
```

Comment:
I personally would have ripped out `derivative` without a replacement, but no strong opinion.
I find it a bit of overkill to pull in an external crate and all this complexity just to avoid writing
```rust
impl Default for Cache {
    fn default() -> Self {
        Self {
            entry_time_to_live: Self::default_entry_time_to_live(),
        }
    }
}
```

But I guess this is a thing of personal preference, so please take this as an FYI ;)

Reply:
No, that makes sense, thank you! :) https://github.com/stackabletech/opa-operator/pull/673/commits/9f49e3c7cb1372eabb016f63d6f4ae7af9a7ce56

Reply:
Thanks for updating!